lefthook.yml

```diff
@@ -3,4 +3,4 @@ pre-commit:
   commands:
     eslint:
       glob: "*.{js,ts,jsx,tsx}"
-      run: yarn lint {staged_files}
+      run: pnpm run lint --dir {staged_files}
```

Comment:
There was same strange issue with the pages directory so I added the ```--dir``` arg. 
Also changed it from using yarn to pnpm so it runs with the proper build too.